### DISCUSSION TITLE
Trakt recomended shows  - Add link to Trakt

### DISCUSSION THREAD

**Discussion by spattersTatt** [pymedusa/Medusa#5660] [Ideas & Feature requests]
Howdy,
a current convert from SickRage over to this platform, and one thing that sickRage has, which medusa does not is when browing the Trakt shows it links to the show in Trakt for further information.  I actually found this usefull and suprised its not currently in medusa....  Can you please consider putting it in?

Currently running:

Medusa Info: 
Branch:                 master
Commit:                 0c0a73583452f988c518adeafa75ee1f17987bdf
Version:                 v0.2.11

Also the IMDB scraper takes along time to load and retrieve any listings, not sure if related, or different library being used in the code, but something to test/check

**Comment by p0psicles** [comment]
This is currently implemented in #5782
